src/main/utils/download.ts

```diff
@@ -16,6 +29,25 @@ export function downloadFileWithProgress(
   let totalBytes = 0;
   const eventsInstance = eventsClient(event);
 
+  const filePath = path.join(localUserPath, 'game.zip');
+
+  // TODO: Optional, prevents from redownloading of the same file
+  if (fileExists(filePath)) {
```

Comment:
jeśli dodamy obsługę zapamiętywania % pobrania i startu od tych % to fileExists będzie do kosza. Niby spoko że coś jest ale dodałbym chociaż todo że to wyleci

Reply:
wahałem się czy to dodać, było mi to użyteczne przy testowaniu, ale nie jestem pewien czy na prod tego nie wyłączyć nawet jak nie będzie zapamiętywania